### ISSUE TITLE
[Feature Request] tf.keras expose align_corners

### ISSUE THREAD

**Issue by bhack** [tensorflow/tensorflow#23041]
----------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
N/A
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
N/A
- **TensorFlow installed from (source or binary)**:
N/A
- **TensorFlow version (use command below)**:
N/A
- **Python version**:
N/A
- **Bazel version (if compiling from source)**:
N/A
- **GCC/Compiler version (if compiling from source)**:
N/A
- **CUDA/cuDNN version**:
N/A
- **GPU model and memory**:
N/A
- **Exact command to reproduce**:
N/A

### Describe the problem
We need to expose `align_corners` in this Keras upsample
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/convolutional.py#L1935

See also https://github.com/keras-team/keras/pull/10994#issuecomment-430305648

**Comment by omalleyt12** [comment]
I can take a look at this, in the meantime you could always use a Lambda layer:

```python
Lambda(
  lambda x: tf.image.resize_bilinear(x, (H, W), align_corners=True), output_shape=(H, W, C))(data))
```

**Comment by bhack** [comment]
@omalleyt12 Do you want to pass trough upstream keras or directly in tf.keras?

**Comment by bhack** [comment]
@ewilderj A classical opensource upstream downstream issue.
We are at risk that users and devs start to duplicate bugs against tf.keras interfaces and on the upstream Keras github repo.

**Comment by bhack** [comment]
@ewilderj I've saw this behavior but I've not annotated the tickets. 
Also in theory general keras API could not be modified troughs direct PR against tf.keras right? 
So I suppose that it is at high risk to become a chicken or eggs problem especially for FR where newcomers start to be exposed to tf.keras directly instead of coming from a Keras experience and I supposed this hypothesis it is not some much remote with the landing of TF 2.0 right?.

**Comment by bhack** [comment]
In the meantime probably could you create a specific `tf.keras` [template](https://github.com/pytorch/pytorch/tree/master/.github)?

**Comment by martinwicke** [comment]
@dksb do you think a separate keras template makes sense to simplify triage?

**Comment by martinwicke** [comment]
@bhack It is definitely possible to modify Keras via PR to tf.keras. We may ask for a mirror PR to keras-team/keras, but we are doing regular syncs, and contributions can move both ways.

**Comment by bhack** [comment]
@martinwicke Probably this topic needs to be better investigated cause we could have a tf.keras api with only 1 backend that break compatibility with Keras API with its own tensorflow backend. More and more users will be exposed to Keras trough tf.keras directly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @omalleyt12: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @omalleyt12: It has been 31 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by bhack** [comment]
@martinwicke @ewilderj This is just another bidirectional case example: https://github.com/tensorflow/tensorflow/issues/24009 VS https://github.com/keras-team/keras/pull/9898#issuecomment-380590161.
Could  we have divergent opinions?
